launchable/commands/split_subset.py

```diff
@@ -60,28 +84,33 @@ def __init__(self, dry_run: bool = False):
             super(SplitSubset, self).__init__(dry_run=dry_run)
 
         def run(self):
-            index = bin_target[0]
-            count = bin_target[1]
-
-            if (index == 0 or count == 0):
-                click.echo(
-                    click.style(
-                        'Error: invalid bin value. Make sure to set over 0 like `--bin 1/2` but set `--bin {}`'.format(
-                            bin_target),
-                        'yellow'),
-                    err=True,
-                )
-                return
-
-            if count < index:
-                click.echo(
-                    click.style(
-                        'Error: invalid bin value. Make sure to set below 1 like `--bin 1/2`, `--bin 2/2` '
-                        'but set `--bin {}`'.format(bin_target),
-                        'yellow'),
-                    err=True,
-                )
-                return
+            if not is_split_by_groups and bin_target is None:
```

Comment:
[ask]
Is it ok to specify `is_split_by_groups` and `bin_target` simultaneously?

Reply:
No, it isn't so I fixed it. Thank you!